poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
If I run `poetry lock --no-update` with Poetry 1.5.1, I am seeing a lot of changes to the lock file. Are you using an older version of Poetry?

Reply:
Thank you for noticing it, it seems that I really had an older version of poetry. Removed the old one, installed the new one, replaced they lock file from main, used poetry lock, now it shall work as expected.